src/main/java/net/imagej/ops/threshold/ThresholdNamespace.java

```diff
@@ -1346,6 +1347,14 @@
 		return result;
 	}
 
+	@OpMethod(op = net.imagej.ops.threshold.thresholdElements.DefaultThresholdElements.class)
```

Comment:
I think this op is really interesting. I would rather call it `FilteringOp` or something like this which takes some `Iterable<O>` and return some other `Iterable<O>` where all `O` fulfill some `Condition<O>`.